pkgs/top-level/all-packages.nix

```
@@ -39436,4 +39436,6 @@ with pkgs;
   gnss-share = callPackage ../servers/gnss-share { };
 
   ali = callPackage ../tools/networking/ali { };
+
+  isolate = callPackage ../os-specific/linux/isolate { };
```

```suggestion
  isolate = callPackage ../tools/misc/isolate { };
```

`os-specific` is for low-level programs that are not made for other environments - things like device drivers, kernel patches...

When in doubt, just throw the things under `tools/misc/`